src/libsync/owncloudpropagator.h

```diff
@@ -210,11 +210,7 @@ class PropagatorCompositeJob : public PropagatorJob
 
     virtual bool scheduleSelfOrChild() Q_DECL_OVERRIDE;
     virtual JobParallelism parallelism() Q_DECL_OVERRIDE;
-    virtual void abort() Q_DECL_OVERRIDE
-    {
-        foreach (PropagatorJob *j, _runningJobs)
-            j->abort();
-    }
+    virtual void abort() Q_DECL_OVERRIDE;
```

Comment:
Actually, the PropagatorJob::abort documentation should state that this function must emit abortFinished